tests/functional/aws-node-sdk/test/multipleBackend/objectTagging/taggingAwsVersioning-putget.js

```diff
@@ -0,0 +1,285 @@
+const async = require('async');
+
+const withV4 = require('../../support/withV4');
+const BucketUtility = require('../../../lib/utility/bucket-util');
+const bucket = 'testawsbackendtaggingversioned';
+
+const { removeAllVersions } = require('../../../lib/utility/versioning-util');
+const {
+    describeSkipIfNotMultiple,
+    awsS3,
+    awsBucket,
+    awsLocation,
+    enableVersioning,
+    putNullVersionsToAws,
+    putVersionsToAws,
+    awsGetLatestVerId,
+    tagging,
+} = require('../utils');
+
+const { putTaggingAndAssert, getTaggingAndAssert, delTaggingAndAssert,
+    awsGetAssertTags } = tagging;
+const someBody = 'teststring';
+
+describeSkipIfNotMultiple('AWS backend object tagging with versioning',
```

Comment:
specify put/get since have separate describe for delete tags?

Reply:
👍